### PR TITLE
10.0 mig base deliver files and documents

### PR DESCRIPTION
**Pull request by mFlayyan**

### CODE REVIEW

**Comment by mFlayyan** [comment]
WIP

**Comment by thomaspaulb** [comment]
@mFlayyan I had done some work on the 8.0 modules [here](https://github.com/OCA/delivery-carrier/pull/163/commits). Could you rewind to `ad0b36e`, then cherry-pick `e839937`, `062b3f7` and `ddcab85` from my PR, and then apply your commits in order? I think they should all work without problems except [this one](https://github.com/OCA/delivery-carrier/pull/184/commits/93366e974a2ca148ab619220b0722d4b59fe5508), which still works with the YML.

**Review by thomaspaulb — CHANGES_REQUESTED** [review]
include work done on 8.0 in PR

File: base_delivery_carrier_files/stock_view.xml [hunk 1]
```diff
@@ -1,33 +1,37 @@
...
-                    </filter>
-                </data>
-            </field>
-        </record>
-    </data>
-</openerp>
+    <record id="view_picking_out_search_carrier_file" model="ir.ui.view">
+        <field name="name">stock.picking.search</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="sale_stock.view_picking_internal_search_inherit"/>
```
Comment (thomaspaulb) [inline base_delivery_carrier_files/stock_view.xml]: This XMLid does not exist in Odoo 10.0, need to set the correct one

File: base_delivery_carrier_files/carrier_file_demo.yml [hunk 2]
```diff
@@ -20,36 +20,45 @@
...
+    picking_type_id: outgoing_shipment_type_carrier_file
     location_dest_id: location_delivery_counter
     carrier_id: delivery.delivery_carrier
 -
  !record {model: stock.move, id: outgoing_shipment_icecream_carrier_file}:
     picking_id: outgoing_shipment_carrier_file
     product_id: product_icecream
     product_uom: product.product_uom_kgm
-    product_qty: 130.0
+    product_uom_qty: 130.0
```
Comment (thomaspaulb) [inline base_delivery_carrier_files/carrier_file_demo.yml:44]: Still seeing some YML files here

File: base_delivery_carrier_files/__manifest__.py [hunk 3]
```diff
@@ -21,9 +21,9 @@
 
 {
     'name': 'Base Delivery Carrier Files',
-    'version': '1.2.3',
+    'version': '10.0.1.0.1',
     'category': 'Generic Modules/Warehouse',
-    'description': """
+    'summary': """
```
Comment (thomaspaulb) [inline base_delivery_carrier_files/__manifest__.py]: This is way too long for a summary :-)

image

**Review by thomaspaulb — CHANGES_REQUESTED** [review]

File: base_delivery_carrier_files/carrier_file_demo.yml (hunk 2 above)
Comment (thomaspaulb) [reply base_delivery_carrier_files/carrier_file_demo.yml:44]: (which also give an error on installing the module)

File: delivery_carrier_file_tnt/__manifest__.py [hunk 4]
```diff
@@ -21,9 +21,9 @@
...
 {
     'name': 'Delivery Carrier File: TNT',
-    'version': '1.0',
+    'version': '10.0.1.0.0',
     'category': 'Generic Modules/Warehouse',
-    'description': """
+    'summary': """
 Sub-module for Base Delivery Carrier Files.
 
 Definition of the delivery carrier file for "TNT Express Shipper".
```
Comment (thomaspaulb) [inline delivery_carrier_file_tnt/__manifest__.py:29]: This summary is still too long, maybe keep only this line.

**Review by thomaspaulb — CHANGES_REQUESTED** [review]

File: base_delivery_carrier_files/__manifest__.py [hunk 5]
```diff
@@ -57,11 +37,11 @@
...
              'wizard/generate_carrier_files_view.xml',
              'security/ir.model.access.csv'],
-    'demo': ['carrier_file_demo.xml',
-             'carrier_file_demo.yml'],
-    'test': ['test/carrier_file.yml',
-             'test/carrier_file_manual.yml'],
+    'demo': ['carrier_file_demo.xml', 'carrier_file_demo.yml'],
+    'test': [ '../account/test/account_minimal_test.xml',
+              'test/carrier_file.yml',
+              'test/carrier_file_manual.yml'],
```
Comment (thomaspaulb) [inline base_delivery_carrier_files/__manifest__.py]: We still should integrate changes from [here](https://github.com/OCA/delivery-carrier/pull/163/files)

**Review by thomaspaulb — CHANGES_REQUESTED** [review]

**Comment by thomaspaulb** [comment]
@mFlayyan Please nick my branch from [here](https://github.com/sunflowerit/delivery-carrier/pull/1), force-push, and do some sanity tests like if the module works well, installs, runs tests etc.

**Comment by thomaspaulb** [comment]
@pedrobaeza This one will not be attended anymore and can be closed in favor of the other one.

**Comment by thomaspaulb** [comment]
Thanks, that was quick

**Pull request closed without merging**